ssh/ssh.go

```diff
@@ -129,6 +130,15 @@ func (r *Agent) Add(id *api.Identity, host string) (err error) {
 	return
 }
 
+//
+// Ensure key formatting.
+func (r *Agent) format(in string) (out string) {
+	if in != "" {
+		out = strings.TrimSpace(in) + "\n"
```

Comment:
Looks good, just a nit, I would think about not adding the "\n" also in case when the _in_ ends with "\n" already.